helpers/database.go

```diff
@@ -162,14 +164,17 @@ func CleanupTestData(ccdb, uaadb *sql.DB, ctx context.Context, testConfig Config
 		"DELETE FROM service_plans WHERE name LIKE '%s'",
 		"DELETE FROM services WHERE label LIKE '%s'",
 		"DELETE FROM service_brokers WHERE name LIKE '%s'",
+		"DELETE FROM events WHERE actee_name LIKE '%s'",
```

Comment:
Duplicate DELETE statements

Reply:
I don't see a duplicate, one delete is for postgres, and the other for mysql. Some tests inserts data with the prefix in the actee_name column, some to the actee column, so both delete statements are needed.

Reply:
Ok, there is a duplicated delete statement, I'm sorry, but it was in line 133